src/cache-control.js

```diff
@@ -1,6 +1,7 @@
-const { config } = require("./config");
+const { getConfig } = require("./config");
```

Comment:
`getConfig` is preferred because the tests programmatically change the `config` object and so wrapping it in a function avoids using a stale config object.